rust/crates/langsmith_tracing_client/src/client/tracing_client.rs

```diff
@@ -0,0 +1,70 @@
+use crate::client::errors::TracingClientError;
+use crate::client::processor::RunProcessor;
+use crate::client::run::QueuedRun;
+use crate::client::run::{RunCreateWithAttachments, RunUpdateWithAttachments};
+use std::time::Duration;
+use tokio::sync::mpsc::{self, Sender};
+use tokio::task::JoinHandle;
+
+pub struct ClientConfig {
+    pub endpoint: String,
+    pub queue_capacity: usize,
```

Comment:
what's the idea with a limited capacity? if it goes over what do we expect should happen